targets/lxqt

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+if [ "${TARGETNOINSTALL:-c}" = 'c' ]
+   if [ "${ARCH#arm}" != "$ARCH" ]; then
+       error 99 "Mate packages are not available for ARM."
```

Comment:
Are these checks correct?  First of all, it's LXQT.  Secondly, some of the text doesn't match the check (Xenial/Trusty).  Thirdly, are you sure there's no ARM builds of LXQT?  It's not like you're adding a repo or anything...